pkg/controller/admissionchecks/multikueue/admissioncheck.go

```diff
@@ -90,28 +99,25 @@ func (a *ACReconciler) Reconcile(ctx context.Context, req reconcile.Request) (re
 				}
 			}
 		}
+		unusableClustersCount := len(missingClusters) + len(inactiveClusters)
+		if unusableClustersCount > 0 {
+			if unusableClustersCount < len(cfg.Spec.Clusters) {
+				// keep it partially active
+				newCondition.Reason = "PartiallyActive"
```

Comment:
I'm not sure we are not introducing a bug in this scenario:
- we have a manager and worker1 and worker2
- the workload "w" is admitted on worker1
- we lose temporarily connection with worker1, we don't want to start the workload on worker2 until the connection loss is permanent.

Could you add a test case regression here?

Reply:
Covered in #1681.

Reply:
I think it is much better to combine these two PRs. 

The issue with this one merged first is that it introduces a bug, that even a temporary loss of connectivity may result in two jobs running at the same time. This is problematic for users who push results to database, and don't want two jobs pushing results at the same time. 

Still, this situation is acceptable, but the time needs to be controllable by users. Let's have one PR which addresses the issue of permanent connection loss, by requeuing of the workloads, where "permanent" is 15min default, but controlled by users.

Reply:
/close 

Moved in #1681